### PR TITLE
use webpack-node-externals to exclude node modules

### PR DESCRIPTION
**Pull request by liady**
Use [webpack-node-modules](https://github.com/liady/webpack-node-externals) to extract node modules from server bundling.
Closes #104

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,7 +59,8 @@
     "json-loader": "0.5.4",
     "just-wait": "1.0.3",
     "webpack": "1.12.11",
-    "webpack-dev-server": "1.14.1"
+    "webpack-dev-server": "1.14.1",
+    "webpack-node-externals": "^0.4.1"
```
Comment (RickWong) [inline package.json]: Could you please remove the caret? 

(Whether the caret is necessary/preferred is out-of-scope of this PR. I'd like to merge it without caret first.)

**Comment by RickWong** [comment]
Thanks @liady. This looks great!

**Comment by liady** [comment]
@RickWong thanks! removed the caret

**Comment by liady** [comment]
Hey, any news on this?

**Pull request merged**